### ISSUE TITLE
Any other way to check without messenger integration

### ISSUE THREAD

**Issue by montaser85** [adeshpande3/Facebook-Messenger-Bot#17]
Is there any way to check what will be the responses for some input sentences without using the messenger. How can we get the responses without 7,8,9 no steps. I am trying to say if you would like to show some simple way to just check what will be the responses if we give some sentences as input.

**Comment by adeshpande3** [comment]
If you take a look at lines 234-243 in Seq2Seq.py, you'll see that every 25 iterations, I'm testing the network on a new test string coming from the list encoderTestStrings. Basically, in line 237, we're calling a function called getTestInput, which converts the given input test you want to test into a vector. Then, in the following lines we just put in zeros for our decoder input and labels, and then we call sess.run in order to compute the decoderPrediction. 

So the description above describes how you can check how the network is doing as the training process is occurring. If you want to test the network after training, you can go ahead and uncomment line 201, remove the training loop, adjust the encoderTestStrings data structure with the input strings you want, and then follow the code in lines 235-243.

**Comment by montaser85** [comment]
To do so, I have uncommented line 202, removed the loop, but I am getting an abnormal reply for sentences which have shown decent reply while training. the code inside the loop looks like this now:
`encoderTrain, decoderTargetTrain, decoderInputTrain = getTrainingBatch(xTrain, yTrain, batchSize, maxEncoderLength)
feedDict = {encoderInputs[t]: encoderTrain[t] for t in range(maxEncoderLength)}
feedDict.update({decoderLabels[t]: decoderTargetTrain[t] for t in range(maxDecoderLength)})
feedDict.update({decoderInputs[t]: decoderInputTrain[t] for t in range(maxDecoderLength)})
feedDict.update({feedPrevious: False})

curLoss, _, pred = sess.run([loss, optimizer, decoderPrediction], feed_dict=feedDict)
	
	
num = randint(0,len(encoderTestStrings) - 1)
print(encoderTestStrings[num])
inputVector = getTestInput(encoderTestStrings[num], wordList, maxEncoderLength);
feedDict = {encoderInputs[t]: inputVector[t] for t in range(maxEncoderLength)}
feedDict.update({decoderLabels[t]: zeroVector for t in range(maxDecoderLength)})
feedDict.update({decoderInputs[t]: zeroVector for t in range(maxDecoderLength)})
feedDict.update({feedPrevious: True})
ids = (sess.run(decoderPrediction, feed_dict=feedDict))
print(idsToSentence(ids, wordList))`

**Comment by adeshpande3** [comment]
Hmm, maybe you could double check to make sure the model is getting loaded in correctly? You can maybe do a sess.run call on some of the variables (like encoderLSTM - although I'm not even sure that will output anything) or just check the models subfolder and make sure your checkpoint and your data and index and meta files are there. 

Also, just to clarify, this issue is occurring whenever you try to load in a pretrained model?
